tests/helpers.bash

```diff
@@ -2,7 +2,7 @@
 
 STORAGE_BINARY=${STORAGE_BINARY:-$(dirname ${BASH_SOURCE})/../containers-storage}
 TESTSDIR=${TESTSDIR:-$(dirname ${BASH_SOURCE})}
-STORAGE_DRIVER=${STORAGE_DRIVER:-vfs}
+STORAGE_DRIVER=${STORAGE_DRIVER:-overlay}
```

Comment:
I'm not okay with changing this testing default -- it trades running tests with the driver that always has to work for running them with another.